renderdoc/data/glsl/debuguniforms.h

```diff
@@ -26,9 +26,11 @@
 // classes that represent a whole cbuffer
 #if defined(__cplusplus)
 
-#include "common/common.h"
-#include "maths/matrix.h"
-#include "maths/vec.h"
+// The Android GL ES shader compiler does not supports the " character.
+// The includes should be before this file is included!
```

Comment:
Yes, it does. I don't know exactly why, but we got compile errors, if we didn't removed the qoutes. And I forgot to remove them from the `#error` directives, I'll fix it soon.